lib/party_foul.rb

```diff
@@ -78,9 +78,9 @@ def self.repo_url
   # Will also setup for GitHub api connections
   #
   # @param [Block]
-  def self.configure(&block)
+  def self.configure
     yield self
-    self.github ||= Octokit::Client.new access_token: oauth_token, api_endpoint: api_endpoint
```

Comment:
I'm not following this one. We surely don't want to instantiate on every request. I see the comments in the commit, but if this is to satisfy a concern in the test suite then there should probably be another workaround considered